README.md

```diff
@@ -3,6 +3,19 @@ Blue Ocean is an alternative user experience for Jenkins. You can learn more abo
 
 Join the community and [![Gitter](https://badges.gitter.im/jenkinsci/blueocean-plugin.svg)](https://gitter.im/jenkinsci/blueocean-plugin?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+# Blue Ocean status
+
+Blue Ocean will not receive further functionality updates.
+Blue Ocean will continue to provide easy-to-use Pipeline visualization, but it will not be enhanced further.
+It will only receive selective updates for significant security issues or functional defects.
+
+Alternative options for Pipeline visualization, such as the [Pipeline: Stage View](https://plugins.jenkins.io/pipeline-stage-view/) and [Pipeline Graph View](https://plugins.jenkins.io/pipeline-graph-view/) plugins, are available and offer some of the same functionality.
```

Comment:
I don't understand why we mention https://plugins.jenkins.io/pipeline-stage-view/.
it's an abandoned plugin.... nobody takes care of it.
BlueOcean has more maintenance...

Reply:
This statement was not written by me. It was written in https://github.com/jenkins-infra/jenkins.io/pull/5288 and I just copied and pasted it here.

Reply:
We wrote that statement in hopes of directing users to alternate ways of visualizing the Pipeline graph.  I wanted to have more than the Pipeline Graph View plugin in the list of alternatives for visualizing Pipeline execution.

Do you have a recommendation for alternate phrasing?

Reply:
To be honest. I don't see any alternatives. BlueOcean is so superrior to anything Jenkins core or any current plugins can provide. To me it is less important to use BlueOcean capabilities as a full replacement of Jenkins UI, but pipeline stages visualisation, jenkins `input` view, and the steps view (console), is mandatory and what currently Jenkins provide is so unintuitive  to use. 

BlueOcean currently in a bad shape (at least the features I listed above), many bugs that require reloading the page, so I understand why the statement is added, and I really hope that one day we will see Jenkins build-in UI will provide something similar to what BlueoOcean can.

Reply:
> We wrote that statement in hopes of directing users to alternate ways of visualizing the Pipeline graph. I wanted to have more than the Pipeline Graph View plugin in the list of alternatives for visualizing Pipeline execution.
> 
> Do you have a recommendation for alternate phrasing?

Alternate phrasing? I'm not sure. just remove pipeline-stage-view at least maybe. It's really abandoned plugin (except if we consider merging `dependabot PRs` as real maintenance). 
I would say just remove `pipeline-stage-view`... BO was build to replace it. (and the statement here could have been written in pipeline-stage-view README as well but giving BO as an alternative ;)). 
It looks like "you're happy using Maven we stop developing it but only simple maintenance. Feel free to return using Ant" 🤣 :trollface: 

I tend to agree on the comment from @shamil  "[let's] be honest, I don't see any alternatives". 

But frankly this state of no new features but only maintenance could be use for a lot of plugins ;). BO is just doing the job. 
I just feel adding such statement is like "we will remove the plugin soon and we do not offer real alternatives"... (I have been asking in such way by some users). Whereas it's an opensource code so can be maintained by anybody and again we have so many plugins in such state (they do the job with no real need of new feature and nobody really maintaining them).
But yeah the goal of this statement is to be clear on the fact nobody came really to help fixing bugs etc.. So better mention the state.